makefile

```diff
@@ -16,18 +23,21 @@ all: $(OUTFILE)
 open: $(OUTFILE)
 	xdg-open "$(OUTFILE)" &>/dev/null & disown
 
-update: $(WEBSITE)/$(OUTFILE)
+update: $(WEBSITE)/$(DIRECTORY)$(OUTFILE)
 
 clean:
 	rm -f "$(OUTFILE)"
 
 $(OUTFILE): $(SOURCES)
 	pandoc --template="$(TEMPLATE)" --highlight-style="${HIGHLIGHT_STYLE}" --standalone --toc --output "$(OUTFILE)" $(SOURCES)
 
-$(WEBSITE)/$(OUTFILE): $(OUTFILE)
-	cp $(OUTFILE) $(WEBSITE)/documents/
+$(OUTFILE_HTML): $(SOURCES)
+	pandoc --filter="$(FILTER_HTML)" --template="$(TEMPLATE_HTML)" --highlight-style="${HIGHLIGHT_STYLE}" --standalone --toc --output "$(OUTFILE_HTML)" $(SOURCES)
+
+$(WEBSITE)/$(DIRECTORY)$(OUTFILE): $(OUTFILE) $(OUTFILE_HTML)
+	cp "$(OUTFILE)" "$(OUTFILE_HTML)" "$(WEBSITE)/$(DIRECTORY)"
 
```

Comment:
Make handles if changes were actually made. Plus git won't make a commit if there's no difference too. That function is a convenience for anyone who has write access to the website repository, which is very few. The branch and upstream are at their discretion, as I've used multiple upstreams and multiple branchs. I understand that someone might make a mistake (as Nick did that one time), but it will always be something that's easy to revert. I don't consider testing the branch and upstream necessary or convenient.